docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -490,14 +502,14 @@ public class ProtectedResource {
     }
 }
 ----
-<1> Request sub-path `/standard-way` requires both resource permission and scope `read` according to the configuration properties we set in the `application.properties` before.
+<1> Request sub-path `/standard-way` requires both resource permission and scope `read` according to the configuration properties we previously set in the `application.properties`.
 <2> Request sub-path `/programmatic-way` only requires permission `Scope Permission Resource`, but we can enforce scope with `SecurityIdentity#checkPermission`.
 <3> The `@PermissionsAllowed` annotation only grants access to the requests with permission `Scope Permission Resource` and scope `read`.
 For more information, see the section xref:security-authorize-web-endpoints-reference.adoc#standard-security-annotations[Authorization using annotations] of the Security Authorization guide.
 
-== Multi-Tenancy
+== Multi-tenancy
 
-It is possible to configure multiple policy enforcer configurations, one per each tenant, similarly to how it can be done for xref:security-openid-connect-multitenancy.adoc[Multi-Tenant OpenID Connect Service Applications].
+You can set up policy enforcer configurations for each tenant, similar to how one does with xref:security-openid-connect-multitenancy.adoc[OpenID Connect (OIDC) multi-tenancy].
```

Comment:
```suggestion
You can set up policy enforcer configurations for each tenant, similar to one with xref:security-openid-connect-multitenancy.adoc[OpenID Connect (OIDC) multi-tenancy].
```

Reply:
I propose `...similar to how it is done with...`

Reply:
Updated. Thank you.